src/analysis/normed_space/pi_Lp.lean

```diff
@@ -71,8 +71,8 @@ def pi_Lp {ι : Type*} (p : ℝ) (α : ι → Type*) : Type* := Π (i : ι), α
 instance {ι : Type*} (p : ℝ) (α : ι → Type*) [∀ i, inhabited (α i)] : inhabited (pi_Lp p α) :=
 ⟨λ i, default⟩
 
-lemma fact_one_le_one_real : fact ((1:ℝ) ≤ 1) := ⟨rfl.le⟩
-lemma fact_one_le_two_real : fact ((1:ℝ) ≤ 2) := ⟨one_le_two⟩
+instance fact_one_le_one_real : fact ((1:ℝ) ≤ 1) := ⟨rfl.le⟩
```

Comment:
Is there any benefit to having this specialized instance, instead of a generic `le_rfl` instance?

Reply:
I can imagine some situations where the system would want to check `fact (p ≤ q)` and then it could try to see if by chance `p = q` and run into a heavy refl computation, while this wouldn't happen if it needed to check whether `p` is `1`. But all this is completely theoretical, I don't have a concrete example in mind. I just went for the smallest possible change.